site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -365,13 +365,21 @@ include::../shared/_advanced-edition-blurb-api.adoc[]
 +
 :advanced_feature!:
 
+[field]#tenant.failedAuthenticationConfiguration.actionCancelPolicy.onPasswordReset# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.42.0#::
+Indicates if you want the user to be able to self-service unlock their account prior to the action duration by completing a password reset workflow.
+
 [field]#tenant.failedAuthenticationConfiguration.actionDuration# [type]#[Long]# [optional]#Optional# [default]#defaults to `3`# [since]#Available since 1.8.0#::
 The duration of the User Action. This value along with the `actionDurationUnit` will be used to set the duration of the User Action. Value must be greater than 0.
 
 [field]#tenant.failedAuthenticationConfiguration.actionDurationUnit# [type]#[String]# [optional]#Optional# [default]#defaults to `"MINUTES"`# [since]#Available since 1.8.0#::
 
 include::_expiry_unit.adoc[]
 
+[field]#tenant.failedAuthenticationConfiguration.emailUser# [type]#[String]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.42.0#::
+Indicates you would like to email the user when the user's account is locked due to this action being taken. This requires the User Action specified by the [field]#tenant.failedAuthenticationConfiguration.userActionId# also to be configured for email. If the User Action is not configured to be able to email the user, this configuration will be ignored. See [field]#userEmailingEnabled# on the User Action.
```

Comment:
```suggestion
Indicates you would like to email the user when the user's account is locked due to this action being taken. This requires the User Action specified by the [field]#tenant.failedAuthenticationConfiguration.userActionId# to also be configured for email. If the User Action is not configured to be able to email the user, this configuration will be ignored. See [field]#userEmailingEnabled# on the User Action.
```

Reply:
Missed this before I merged. Will make this change thanks!